test/IECore/PointsAlgoTest.py

```diff
@@ -283,6 +283,143 @@ def testPrimitiveVariablesCorrectlyFiltered(self):
 		self.assertEqual( points["e"].data, IECore.FloatVectorData( range( 0, 10, 2 ) ) )
 		self.assertEqual( points["e"].interpolation, IECore.PrimitiveVariable.Interpolation.FaceVarying)
 
+class MergePointsTest( unittest.TestCase ) :
+
+	#todo find a better place for this decorator to live
+	#decorating a testfunction gives me time & pid to connect gdb to look at test failures.
+	def debugger(fn):
```

Comment:
In some other modules, we'd define a [`TestCase`](https://github.com/GafferHQ/gaffer/blob/master/python/GafferTest/TestCase.py#L50) base class. I was surprised we don't have one in IECore yet though.

Reply:
Ah, IECoreRI has one, but it doesn't live where one might think. https://github.com/ImageEngine/cortex/blob/master/python/IECoreRI/TestCase.py. So maybe something like that in IECore?